osu.Game/Screens/OsuScreen.cs

```diff
@@ -73,6 +73,20 @@ protected override IReadOnlyDependencyContainer CreateChildDependencies(IReadOnl
             return deps;
         }
 
+        /// <summary>
+        /// Unbind and return leases for all <see cref="Bindable{T}"/>s managed by the exiting screen.
+        /// </summary>
+        /// <remarks>
+        /// While all bindables will eventually be returned by disposal logic, this is too late as
+        /// leases would be in a leased state during exiting transitions.
+        /// This localised handling ensures a correct leased state before <see cref="IScreen.OnResuming"/>.
+        /// </remarks>
+        public void UnbindAll()
```

Comment:
`virtual`.

Reply:
On further discussion, we agreed to move this logic completely to the framework side (`ScreenStack` should call the internal `UnbindAllBindables`).